Apps/WebClient/src/appsettings.json

```diff
@@ -156,5 +156,8 @@
             "RedirectUri": "myhealthbc://*"
         },
         "Version": 2
+    },
+    "ClientRegistry": {
```

Comment:
Does WebClient need this config? I haven't seen any code that uses it in this project?